mod/c_members_byDepartment/pages/members/members_by_dept.php

```diff
@@ -37,7 +37,8 @@
 $provinces['pov-sask'] = 'Government of Saskatchewan';
 $provinces['pov-yuk'] = 'Government of Yukon';
 $departmentsEn = array_merge($departmentsEn,$provinces);
-
+unset($departmentsEn['ou=INAC-AANC, o=GC, c=CA']);
```

Comment:
Is there a reason we are not adding the two new departments to this page?

Reply:
Missed it on this page. Added them in